configs/animal_2d_keypoint/topdown_heatmap/animalpose/hrnet_w32_animalpose_256x256.py

```diff
@@ -0,0 +1,123 @@
+_base_ = [
+    '../../../_base_/default_runtime.py',
+    '../../../_base_/schedules/schedule_bs512_ep210.py',
```

Comment:
Per the discussion today, we will keep the schedule settings in the model's config file, instead of a base file.

Reply:
Done. 

BTW, `train_cfg`, `val_cfg`, and `test_cfg` are moved to `default_runtime.py`. Only `train_cfg.max_epochs` is shown in model's config file.

Reply:
Maybe show `train_cfg.val_interval` in the model's config too?

Reply:
OK